bpf/bpf_host.c

```diff
@@ -1127,4 +1132,120 @@ int to_host(struct __ctx_buff *ctx)
 	return ret;
 }
 
+#if defined(ENABLE_HOST_FIREWALL) && !defined(ENABLE_ROUTING)
```

Comment:
The `!defined(ENABLE_ROUTING)` is only relevant for ep routes? Or also tunneling and other modes?

Reply:
I'm probably confused given we also have `ENABLE_ENDPOINT_ROUTES`?

Reply:
> The !defined(ENABLE_ROUTING) is only relevant for ep routes? Or also tunneling and other modes?

Only for endpoint routes AFAIK. `ENABLE_ENDPOINT_ROUTES` was introduced latter (even though I pointed to `ENABLE_ROUTING` at the time). We could probably consolidate in favor of `ENABLE_ENDPOINT_ROUTES`.

Reply:
That would be great given the ever increasing complexity of our ifdef workarounds. :/